storage/aptosdb/src/db/include/aptosdb_reader.rs

```diff
@@ -724,6 +725,50 @@ impl DbReader for AptosDB {
             self.state_store.get_usage(version)
         })
     }
+
+    fn get_db_tailor_iter(
```

Comment:
Looking at the existing code, state sync, backup service, and probably other stuff, they all zip the iterators on the call site -- I still thing this doesn't belong to DbReader.